battery.py

```diff
@@ -0,0 +1,67 @@
+import os
+import power
+
+battery_color_good = [  0,230,0]
+battery_color_ok   = [255,215,0]
+battery_color_bad  = [255,  0,0]
```

Comment:
maybe use tuples for colors, since those don't need to be changed while running

Reply:
yes, this is just plainly borrowed from the [digital clock](https://badge.team/projects/digitalclock) egg, so could be improved to be tuples